qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulationSymmetriseTabModel.h

```diff
@@ -29,10 +29,11 @@ class MANTIDQT_INELASTIC_DLL InelasticDataManipulationSymmetriseTabModel {
   void setEMin(double value);
   void setEMax(double value);
   void setIsPositiveReflect(bool value);
+  bool getIsPositiveReflect();
```

Comment:
I think this function can be const
```suggestion
  bool getIsPositiveReflect() const;
```